docs.joern.io/content/frontends/_index.md

```diff
@@ -3,4 +3,30 @@ id: frontends
 title: Frontends
 bookCollapseSection: true
 weight: 135
----
\ No newline at end of file
+---
+
+## Frontends
+
+Joern contains multiple frontends, each responsible for parsing the source code and generating the AST for their respective languages. `X2CPG` is the base on which all of the other frontends are built, and contains common structures by all of the languages. Each frontend contains the base arguments from `x2cpg`, as well as custom arguments that is frontend specific. See each respective language's page for a detailed explanation on what arguments are available and what the arguments do.
```

Comment:
`language's` => `languages'`

`what the arguments do.` => `what each of the arguments do.`